encoding/rs/encode.go

```diff
@@ -127,3 +143,36 @@ func (g *Encoder) ExtendPolyEval(coeffs []fr.Element) ([]fr.Element, []fr.Elemen
 
 	return evals, pdCoeffs, nil
 }
+
+type JobRequest struct {
+	Index      uint64
+	FrameIndex uint64
+}
+
+func (g *Encoder) interpolyWorker(
+	polyEvals []fr.Element,
+	jobChan <-chan JobRequest,
+	results chan<- error,
+	frames []Frame,
+) {
+
+	for jr := range jobChan {
+		i := jr.Index
+		k := jr.FrameIndex
+		j := rb.ReverseBitsLimited(uint32(g.NumChunks), uint32(i))
+		ys := polyEvals[g.ChunkLength*i : g.ChunkLength*(i+1)]
+		err := rb.ReverseBitOrderFr(ys)
+		if err != nil {
+			results <- err
```

Comment:
should have `continue` here

Reply:
Good catch. I used "return", because if anything that has error, the entire MakeFrames becomes false.

Reply:
I think it's better to use `continue`. If you had enough errors for all of the workers to return, the program could hang on L104.

Continuing ensures that the workers continue to consume the requests. 

Obviously, you could optimize this further if an error was likely here, but this would be a programming error so I don't think it needs to be optimized.

Reply:
I see. good call.